user-service/src/main/java/rs/edu/raf/banka1/controller/OrderController.java

```diff
@@ -5,25 +5,18 @@
 import io.swagger.v3.oas.annotations.media.Schema;
 import io.swagger.v3.oas.annotations.responses.ApiResponse;
 import io.swagger.v3.oas.annotations.responses.ApiResponses;
+import org.springframework.http.HttpStatus;
 import org.springframework.http.MediaType;
 import org.springframework.http.ResponseEntity;
 import org.springframework.security.access.prepost.PreAuthorize;
 import org.springframework.web.bind.annotation.*;
-import rs.edu.raf.banka1.dtos.LoanDto;
-import rs.edu.raf.banka1.dtos.LoanFullDto;
-import rs.edu.raf.banka1.dtos.LoanRequestDto;
-import rs.edu.raf.banka1.model.LoanRequestStatus;
 import rs.edu.raf.banka1.model.OrderStatus;
-import rs.edu.raf.banka1.requests.CreateLoanRequest;
-import rs.edu.raf.banka1.requests.CreateUserRequest;
 import rs.edu.raf.banka1.requests.StatusRequest;
 import rs.edu.raf.banka1.requests.order.CreateOrderRequest;
-import rs.edu.raf.banka1.services.LoanService;
+import rs.edu.raf.banka1.responses.ChangeOrderResponse;
 import rs.edu.raf.banka1.services.OrderService;
 import rs.edu.raf.banka1.utils.JwtUtil;
 
-import java.util.List;
-
 @RestController
 @CrossOrigin
 @RequestMapping("/orders")
```

Comment:
Da li si ažurirao putanje u **endpoints** fajlu? Ako nisi, to bi trebalo da uradiš, da bi se sve poklapalo sa onim što piše u fajlu.

Reply:
Nisam jos, nesto sam dodovao, ali cekam konacne verzije metoda pa cu da dodam.

Reply:
Dobro.